media-sound/guitarix/guitarix-0.44.1.ebuild

```diff
@@ -62,8 +62,9 @@ BDEPEND="
 DOCS=( changelog README )
 
 PATCHES=(
-	"${FILESDIR}"/${PN}-0.41.0-nostrip.patch
-	"${FILESDIR}"/${PN}-0.41.0-py3.11.patch
+	"${FILESDIR}"/${P}-nostrip.patch
```

Comment:
I usually only do ${P} if a patch is upstream, as a hint to drop it next time.

Reply:
Yeah, good point. I'll revert that.